.storybook/preview-head.html

```diff
@@ -2,9 +2,9 @@
   window.global = window;
 </script>
 
-<!-- <style>
-  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;700&display=swap');
-</style> -->
+<style>
```

Comment:
I inspected the typography storybook story and the Inter font wasn't being loaded for storybook, @pietrograndi @moebiusmania do you know if this was commented for some reason?

Reply:
nope, I guess I was experimenting with loading it in different ways and for some reason didn't removed one of the comments

Reply:
ok thanks